src/Microsoft.ML.Data/Evaluators/Metrics/MulticlassClassificationMetrics.cs

```diff
@@ -82,9 +82,15 @@ public sealed class MulticlassClassificationMetrics
         /// </remarks>
         public IReadOnlyList<double> PerClassLogLoss { get; }
 
-        internal MulticlassClassificationMetrics(IExceptionContext ectx, DataViewRow overallResult, int topKPredictionCount)
+        /// <summary>
+        /// The <a href="https://en.wikipedia.org/wiki/Confusion_matrix">confusion matrix</a> giving the counts of the
```

Comment:
>confusion matrix [](start = 73, length = 16)

in BinaryClassificationMetrics you use:`2 dimentional confusion matrix`
is this one k-dimensional?